src/seedu/addressbook/ui/TextUi.java

```diff
@@ -17,34 +17,24 @@
  */
 public class TextUi {
 
-    /** A decorative prefix added to the beginning of lines printed by AddressBook */
-    private static final String LINE_PREFIX = "|| ";
-
-    /** A platform independent line separator. */
-    private static final String LS = System.lineSeparator();
-
-    private static final String DIVIDER = "===================================================";
-
-    /** Format of indexed list item */
-    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
-
-
     /** Offset required to convert between 1-indexing and 0-indexing.  */
     public static final int DISPLAYED_INDEX_OFFSET = 1;
-
+    private static final String DIVIDER = "===================================================";
     /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
     private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
 
```

Comment:
Perhaps more formatting-related stuff (e.g. `COMMENT_LINE_FORMAT_REGEX`) can be moved to the `Formatter`? May be even things like this should be moved to the Formatter? Note that it may not always be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.